Formula/o/onnxnostaticreg.rb

```diff
@@ -0,0 +1,98 @@
+class Onnxnostaticreg < Formula
+  desc "Open standard for machine learning interoperability (built for onnxruntime)"
+  homepage "https://onnx.ai/"
+  url "https://github.com/onnx/onnx/archive/refs/tags/v1.17.0.tar.gz"
+  sha256 "8d5e983c36037003615e5a02d36b18fc286541bf52de1a78f6cf9f32005a820e"
+  license "Apache-2.0"
+
+  depends_on "cmake" => [:build, :test]
+  depends_on "abseil"
+  depends_on "protobuf"
+
+  uses_from_macos "python" => :build
+
+  def install
+    args = %W[
+      -DBUILD_SHARED_LIBS=ON
+      -DCMAKE_INSTALL_RPATH=#{rpath}
+      -DONNX_DISABLE_STATIC_REGISTRATION=ON
```

Comment:
Can we just rebuild `onnx` with this configuration instead?

Reply:
We can, and it will build, but any program linking with onnx only will not work correctly, since it will skip over the schema registration at startup.